controllers/admin/AdminPaymentPreferencesController.php

```diff
@@ -258,7 +258,7 @@ public function renderView()
                           'identifier' => 'id_country',
                           'icon' => 'icon-globe',
                     ),
-                    array('items' => Carrier::getCarriers($this->context->language->id),
+                    array('items' => Carrier::getCarriers($this->context->language->id, false, false, false, null, (defined('ALL_CARRIERS') ? ALL_CARRIERS : null)),
```

Comment:
we shouldn't have here ALL_CARRIERS but Carrier::ALL_CARRIERS, and we don't need to check if that's defined